src/main/java/org/apache/sysds/runtime/instructions/cp/QuantilePickCPInstruction.java

```diff
@@ -127,4 +130,22 @@ public void processInstruction(ExecutionContext ec) {
 				throw new DMLRuntimeException("Unsupported qpick operation type: "+_type);
 		}
 	}
+
+	public OperationTypes getOperationType() {
+		return _type;
+	}
+
+	public boolean isInMem() {
+		return _inmem;
+	}
+
+	@Override
+	protected FEDInstruction tryReplaceWithFederated(ExecutionContext ec) {
+		// Should we federate if the quantiles are given as a federated matrix? (input2)
+		// This might be useful for privacy and for performance just in extreme corner cases.
+		// The current implementation does not support it at the moment.
```

Comment:
This comment could be removed as we anyway only replace instructions where equivalent FED implementations exist, but it is worth discussing/remembering what to do about this case.